src/modules/drawing/DrawingModule.vue

```diff
@@ -1,50 +1,193 @@
 <template>
-    <div v-if="show" class="draw-overlay">
-        <DrawingToolbox
-            :drawing-modes="drawingModes"
-            :current-drawing-mode="currentDrawingMode"
-            @close="hideDrawingOverlay"
-            @setDrawingMode="changeDrawingMode"
-        />
+    <div>
+        <div v-if="show" class="draw-overlay">
+            <DrawingToolbox
+                :drawing-modes="drawingModes"
+                :current-drawing-mode="currentDrawingMode"
+                @close="hideDrawingOverlay"
+                @setDrawingMode="changeDrawingMode"
+            />
+        </div>
+        <div v-show="show">
+            <drawing-style-popup
```

Comment:
While it is correct to use lowercase kebab-case component names, I prefer that we use the UpperCase notation whenever possible, to reflect the name of the `import ...` in the JS.